### PR TITLE
Convert neuron (and spinal) project from TrueSTUDIO to CubeIDE

### PR DESCRIPTION
**Pull request by greenpepper123**
Converted neuron_f4 as shown in PR title.
It works (at least blue indicator LED is blinking after writing newly built firmware).

### CODE REVIEW

File: aerial_robot_nerve/spinal/mcu_project/TrueSTUDIO/spinal/spinal_converter.log [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Project: spinal
+Converter: Atollic TrueSTUDIO project converter
+Date: 20210414
```
Comment (tongtybj) [inline aerial_robot_nerve/spinal/mcu_project/TrueSTUDIO/spinal/spinal_converter.log:3]: I think we can add this log file into .gitignore.

Comment (greenpepper123) [reply aerial_robot_nerve/spinal/mcu_project/TrueSTUDIO/spinal/spinal_converter.log:3]: I know, I just wanted this log to be included during the migration. I'll add this log (and also debug launch file, maybe) into .gitignore.

**Comment by tongtybj** [comment]
@greenpepper123 

did you add commit related to your B4's work, such as hydrus_xi opening the door? please check the commit logs

**Comment by greenpepper123** [comment]
Yes, I did it on purpose (for test convenience), but I didn't noticed that it would be added to this PR.
I'll create another branch for develop and revert some commits in `rtos` branch.

**Comment by tongtybj** [comment]
This is merge by #483

**Pull request closed without merging**